frontend/src/components/ActiveValidationModal.js

```diff
@@ -111,7 +109,7 @@ const ActiveValidationModal = ({
     if (caught) {
       e.preventDefault();
     }
-  }, [index, images, submittable, model, toggleLabel]);
+  }, [index, images, submittable, model, toggleLabel, setIsOpen]);
```

Comment:
This is a valid dependency of the hook and can be added safely without any side-effect